source/install/yum.rst

```diff
@@ -43,6 +43,12 @@ It is recommended to intall |PSMDB| from official Percona repositories:
    `Percona Software Repositories Documentation
    <https://www.percona.com/doc/percona-repo-config/index.html>`_.
 
+#. Enable the repository.
+   
+   .. code-block:: bash
+   
+      $ sudo percona-release enable psmdb-36 
+
 #. Install the required |PSMDB| package using :command:`yum`.
    For example, to install the full package, run the following:
 
```

Comment:
In install section please change Percona-Server-MongoDB-34 to Percona-Server-MongoDB-36